tutorials/io/plot_30_reading_fnirs_data.py

```diff
@@ -37,6 +37,22 @@
              stored in the .snirf format.
 
 
+.. _import-boxy:
+
+BOXY (.txt)
+================================
+
+BOXY recordings can be read in using :func:`mne.io.read_raw_boxy`.
+The BOXY software and ISS Imagent I and II devices store data in a single .txt
+file containing DC (overall background light intensity),
+AC (modulated light intensity), and Phase information for each source and
+detector combination. These raw data files can be saved as parsed or unparsed
+.txt files, which affects how the data in the file is organised.
+MNE will read either file type and extract the raw DC, AC, and Phase data.
+If triggers are sent using the 'digaux' port of the recording hardware, MNE
+will also read the 'digaux' data and create annotations for any triggers.
+
```

Comment:
I think this is a good place to make a note about why the FD reader returns a CW type.